test/unit/normalize.js

```diff
@@ -20,10 +20,8 @@ describe('Normalization', function () {
   it('spreads rgb image values between 0 and 255', function (done) {
     sharp(fixtures.inputJpgWithLowContrast)
       .normalise()
-      .raw()
-      .toBuffer(function (err, data, info) {
+      .toFile('test2.jpg', function (err) {
```

Comment:
Was this change intentional?

Reply:
no this was an accident i have fixed it.